test/components/task-plan/footer.spec.coffee

```diff
@@ -57,7 +57,6 @@ VISIBLE_HW = extendBasePlan({
 
 helper = (model) ->
   {id} = model
-  console.info(model)
```

Comment:
that actually came from your work last night I think.  And I didn't notice when reviewing either...